modules/antitracking/sources/steps/oauth-detector.es

```diff
@@ -127,15 +127,20 @@ export default class OAuthDetector {
   /**
    * Pipeline step to check if this request is part of a OAuth flow. This is done by
    * checking that the following three conditions are met:
-   *  - The third-party url path contains '/oauth'
+   *  - The third-party url path contains '/oauth' or '/authorize'
    *  - The user has recently clicked in the source tab (i.e. the site wanting to authenticate the
    * user)
    *  - The user has recently visited the oauth domain (the authentication provider)
    * @param state
    * @returns false if the request is an oauth request, true otherwise
    */
   checkIsOAuth(state) {
-    if (state.urlParts.path.indexOf('/oauth') > -1 &&
+    const oAuthUrls = ['/oauth', '/authorize'];
+    const mapper = (oAuthUrl) => state.urlParts.path.indexOf(oAuthUrl) > -1;
```

Comment:
The style guide expects no parenthesis around single-argument arrow functions with no brackets. https://github.com/airbnb/javascript#arrows--one-arg-parens

Reply:
👍 thanks, pushed another commit.